tools/wptrunner/wptrunner/browsers/chrome_ios.py

```diff
@@ -45,37 +43,13 @@ def env_options():
     return {}
 
 
-class ChromeiOSBrowser(Browser):
+class ChromeiOSBrowser(WebDriverBrowser):
     """ChromeiOS is backed by CWTChromeDriver, which is supplied through
     ``wptrunner.webdriver.CWTChromeDriverServer``.
     """
 
     init_timeout = 120
 
-    def __init__(self, logger, webdriver_binary, webdriver_args=None, **kwargs):
-        """Creates a new representation of Chrome."""
-        Browser.__init__(self, logger)
-        self.server = CWTChromeDriverServer(self.logger,
-                                            binary=webdriver_binary,
-                                            args=webdriver_args)
-
-    def start(self, **kwargs):
-        self.server.start(block=False)
-
-    def stop(self, force=False):
-        self.server.stop(force=force)
-
-    def pid(self):
-        return self.server.pid
-
-    def is_alive(self):
-        # TODO(ato): This only indicates the driver is alive,
-        # and doesn't say anything about whether a browser session
-        # is active.
-        return self.server.is_alive()
-
-    def cleanup(self):
-        self.stop()
-
-    def executor_browser(self):
-        return ExecutorBrowser, {"webdriver_url": self.server.url}
+    def make_command(self):
+        return [self.binary,
+                "--port=%s" % str(self.port)] + self._args
```

Comment:
```suggestion
                f"--port={self.port}"] + self._args
```

That probably allow rewrapping too.